aws/rust-runtime/aws-hyper/Cargo.toml

```diff
@@ -23,7 +23,7 @@ aws-smithy-types = { path = "../../../rust-runtime/aws-smithy-types" }
 bytes = "1"
 fastrand = "1.4.0"
 http = "0.2.3"
-http-body = "0.4.0"
+http-body = "0.4.4"
```

Comment:
do we have to be specific here or could we do `"0.4"`? Did we ever write down guidelines for how we specify dependency versions?

Reply:
we can't do 0.4—we are not compatible with `0.4.0` (that's the entire genesis of this PR)